src/data/pnat/factors.lean

```diff
@@ -8,6 +8,18 @@ import data.multiset.sort
 import data.int.gcd
 import algebra.group
 
+/-!
+# Prime factors of nonzero naturals
+
+This file defines the p-adic valuations of a nonzero natural number `n` as a multiset of primes,
```

Comment:
```suggestion
This file defines the factorization of a nonzero natural number `n` as a multiset of primes,
```

Reply:
No worries! I can always write more docstrings in the meantime 😁